### ISSUE TITLE
Zone costs are not summed correctly

### ISSUE THREAD

**Issue by cfe316** [GenXProject/GenX#63]
Possible issue in the costs.csv output: 

There seems to be an issue with adding up the zone-specific costs. When I run the SmallNewEngland/OneZone example (with all default settings, on the current `main` branch `935f19efce8afe7344efe615bf2a19c39c68883c` or on the Dev branch `aa6fd65e063ac1f3b5d7677b682671a80e43d214`) I get
```
Costs,      Total, Zone1
cTotal,    2.46e10, 1.46e10
cFix,      2.46e10, 2.46e10
cVar,       1.39e7,  1.39e7
cNSE,0.0,0.0
cStart,0.0,0.0
cUnmetRsv,0.0,-
cNetworkExp,0.0,-
```
(some decimals have been truncated and formatted to protect the innocent)
The Zone1 cost total is lower than the fixed cost, which seems wrong. It does look like the (`Total`, `cTotal`) cost is the correct summation.

Wilson was able to reproduce this output.

**Comment by xuqingyu** [comment]
This is fixed, right?

**Comment by cfe316** [comment]
No.

**Comment by xuqingyu** [comment]
OK, check my branch, I fixed there.

**Comment by xuqingyu** [comment]
but you are right, I identified the same issue and I fixed it. If my branch is too complex to implemented, it is pretty easy to fix.

**Comment by cfe316** [comment]
I think this can be fixed after #157 .

**Comment by xuqingyu** [comment]
True. Let's wait on that.

**Comment by sambuddhac** [comment]
On to testing #157, after wrapping up #151

**Comment by dharik13** [comment]
@cfe316 @sambuddhac can we incorporate this fix into the latest merge? A few more people have asked me about it and its a relatively straightforward fix

**Comment by sambuddhac** [comment]
> @cfe316 @sambuddhac can we incorporate this fix into the latest merge? A few more people have asked me about it and its a relatively straightforward fix
> 

Just merged after a routine test

**Comment by sambuddhac** [comment]
Just merged

**Comment by cfe316** [comment]
This has now been fixed in 0.3.
